lib/api/apiUtils/object/continueToken.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * generateToken - generates obfuscated continue token from object keyName
+ * @param {String} keyName - name of key to obfuscate
+ * @return {String} - obfuscated continue token
+ */
+function generateToken(keyName) {
+    if (keyName === '' || keyName === undefined) {
+        return undefined;
+    }
+    return Buffer.from(keyName).toString('base64');
```

Comment:
if we pass an empty string to `Buffer.from(keyName).toString('baset64')`, it will be able to return an empty string. On that note, if lines 8 and 20 in this file return `''` instead of `undefined`, line 182 in `lib/api/bucketGet.js` will not need the `||` conditional.